### PR TITLE
Support nested state

### PR DESCRIPTION
**Pull request by lozlow**

### CODE REVIEW

**Comment by swagatha-christie** [comment]
By analyzing the blame information on this pull request, we identified @jhollingworth and @gschambers to be potential reviewers

File: src/createCollection.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import createReducer from './createReducer'
 
 export const REQUEST_TIMEOUT = 10000 // 10s
 
-export default function createCollection (name, createUrl, isResponseValid = _.isArray) {
+export default function createCollection (name, createUrl, path, isResponseValid = _.isArray) {
```
Comment (jhollingworth) [inline src/createCollection.js]: should `path` have a default param?

Comment (jhollingworth) [inline src/createCollection.js]: and should we have it after `isResponseValid`?

Comment (jhollingworth) [inline src/createCollection.js]: should have made it an object literal...

**Pull request merged**